.gitignore

```diff
@@ -3,3 +3,9 @@
 test/report/*.html
 test/report/*.json
 *.log
+.DS_Store
+/samples/.DS_Store
+templates/.DS_Store
+templates/_common/.DS_Store
+test/.DS_Store
+test/features/.DS_Store
```

Comment:
All those should be on your global git configuration, not in the repository.
It does not make sense to start ignoring leftovers made by each operating system or IDE.